content/efcore/getting-started/samples/loading-single-entities/Program.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        using (var context = new LibraryContext())
+        {
+            //grabs the Title with a BookId of 1
```

Comment:
It grabs the Book, not the Title. It prints the title afterwards.

I don't think this comment is necessary, though. The code is self-documenting.